### PR TITLE
Add pinch-to-zoom

### PR DESCRIPTION
**Pull request by micronen**

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @micronen, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;companion.home-assistant#683).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: docs/integrations/android-webview.md [hunk 1]
```diff
@@ -35,5 +35,8 @@ The ![Android](/assets/android.svg) Android app has the ability to keep screen o
 
 This feature may also be controlled by Notification command, [see details](https://companion.home-assistant.io/docs/notifications/notification-commands#screen-on).
 
+## Pinch To Zoom
+The ![Android](/assets/android.svg) Android app has the ability to enable Pinch-To-Zoom to allow multi-touch zoom by enabling corresponding setting in Companion App Configuration.
```
Comment (dshokouhi) [inline docs/integrations/android-webview.md:39]: Can you please add a beta label next to the Android logo? You can reference a recent PR for an example of the tag to use.

**Review by dshokouhi — CHANGES_REQUESTED** [review]
Just needs to be marked as a beta feature

File: docs/integrations/android-webview.md (hunk 1 above)
Comment (dshokouhi) [reply docs/integrations/android-webview.md:39]: @micronen just a FYI we need to add the beta tag here in case you missed this notification :)

Comment (micronen) [reply docs/integrations/android-webview.md]: Added beta tag, as found elsewhere in the docs

**Review by dshokouhi — APPROVED** [review]
Looks good to me, can be merged once parent is merged

**Pull request merged**